oci/zookeeper/image.yaml

```diff
@@ -5,7 +5,7 @@ upload:
     commit: 1781da1dacbacf647319731ff25ba4ad00082140
     directory: .
     release:
-      3.8.4-22.04:
+      3.8-22.04:
```

Comment:
It is probably better to skip the patch version

Reply:
btw, now it is already published. How do you want to handle it?

Reply:
can we remove it? I suppose we should since it is also wrong. The branch has a name that does not correspond to the actual workload version

Reply:
this can be tricky. since it's a recently pushed tag, we can try and hope there are no complaints on the user side.

For that, please update https://github.com/canonical/oci-factory/blob/main/oci/zookeeper/_releases.json#L2-L7. I'll take care of removing the tags on the registry side

Reply:
Yes, I **really** don't want this to be an habit. I don't think there should be any problem to do it in a specific case, given it is a super super fresh version and track. I agree with you if the image had already multiple pulls. BTW, can we check the pulls of a specific hash in dockerhub as much as we can do in GHCR?

Reply:
through parsing of CSV files, yes